tests/kernel/wave/wave_gemm_test.py

```diff
@@ -355,7 +358,12 @@ def repeat(acc: tkl.Register[M, N, tkl.f32]) -> tkl.Register[M, N, tkl.f32]:
 @pytest.mark.parametrize("shape", get_test_shapes("test_gemm"))
 @pytest.mark.parametrize(
     "enable_scheduling",
-    [SchedulingType.NONE, SchedulingType.PREFETCH, SchedulingType.MODULO],
+    [
+        SchedulingType.NONE,
+        _xfail(SchedulingType.PREFETCH),
```

Comment:
If I am reading this right, previously with the smaller tile size, prefetch was working correctly. So is prefetch failing because of the larger tile size or the combination of the larger tile size and the contents of this PR?

Reply:
It doesn't fail if I disable `in-thread-transpose`

Reply:
So are we losing some functionality here? (a test that used to pass, no longer does?)